testnets/epixtestnet/assetlist.json

```diff
@@ -0,0 +1,36 @@
+{
+  "$schema": "../../assetlist.schema.json",
+  "chain_name": "epixtestnet",
+  "assets": [
+    {
+      "description": "The EPIX coin, the lifeblood of the Epix network, providing the utility that powers every interaction and transaction within the ecosystem.",
+      "denom_units": [
+        {
+          "denom": "atepix",
+          "exponent": 0
+        },
+        {
+          "denom": "tepix",
+          "exponent": 18
+        }
+      ],
+      "base": "atepix",
+      "name": "Epix Testnet",
+      "display": "tepix",
+      "symbol": "TEPIX",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/epixtestnet/images/epix.png",
```

Comment:
The images being uploaded to the epixtestnet/images/ directory are identical to the mainnet images, so we don't need the duplicates. Please use image_sync and point to the mainnet asset, and update the path to the mainnet images. E.g.,
```
  "image_sync": {
    "chain_name": "epix",
    "base_denom": "aepix"
  },
  "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/epix/images/epix.png",
  "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/epix/images/epix.svg"
```

Reply:
Please remove the testnets/epixtestnet/images/ directory one all references to it have been updated.

Reply:
Thanks for the suggestions, completed these changes.